spec/form/work_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Sufia::Forms::WorkForm do
```

Comment:
ok so this is confusing, the form I am testing is work_form and when I use your suggestion, I get an  error: uninitialized constant Sufia::Forms::GenericWorkForm (NameError)